### DISCUSSION TITLE
link to documentation of relevant rust functions

### DISCUSSION THREAD

**Discussion by szabgab** [rust-lang/rust-clippy#12234] [Ideas]
Reading about [create_dir](https://rust-lang.github.io/rust-clippy/master/index.html#/create_dir)

Now I'd like to read about [create_dir](https://doc.rust-lang.org/std/fs/fn.create_dir.html) and [create_dir_all](https://doc.rust-lang.org/std/fs/fn.create_dir_all.html) and understand the differences.  IMHO It would be nice if there were links to these pages.

It might be also nice to include a short explanation what is the difference between the two so readers won't need to read all the docs, but regardless the links would be useful.

**Comment by flip1995** [comment]
We are happy to accept a PR that adds this addition links and information!
